numpy/core/_add_newdocs.py

```diff
@@ -1656,6 +1659,20 @@
 
     In such cases, the use of `numpy.linspace` should be preferred.
 
+    The built-in :py:class:`range` generates :std:doc:`Python built-in integers
+    that have arbitrary size <c-api/long>`, while `numpy.arange` produces
+    `numpy.int32` or `numpy.int64` numbers. This may result in incorrect
+    results for large integer values::
+
+      >>> power = 40
+      >>> modulo = 10000
+      >>> x1 = [(n ** power) % modulo for n in range(8)]
+      >>> x2 = [(n ** power) % modulo for n in np.arange(10)]
```

Comment:
```suggestion
      >>> x1 = [(n ** power) % modulo for n in range(8)]
      >>> x2 = [(n ** power) % modulo for n in np.arange(8)]
```
I think making these values the same makes the example a little more digestible, otherwise users may wonder why they are different.

Reply:
OOps yes I was actually looking to get it all to fit nicely into the text box